model/host/host.go

```diff
@@ -999,7 +999,7 @@ func FindHostsSpawnedByBuild(buildID string) ([]Host, error) {
 
 func FindTerminatedHostsRunningTasks() ([]Host, error) {
 	hosts, err := Find(db.Query(bson.M{
-		StatusKey: bson.M{"$nin": evergreen.UphostStatus},
+		StatusKey: bson.M{"$nin": evergreen.RunningHostStatuses},
```

Comment:
Could this just be `StatusKey: evergreen.HostTerminated`?

Reply:
yep, that's a lot better